packages/front-end/components/Experiment/EditMetricsForm.tsx

```diff
@@ -36,12 +39,31 @@ const EditMetricsForm: FC<{
   const filteredMetrics = metricDefinitions.filter(
     (m) => m.datasource === datasource?.id
   );
+
+  const defaultMetricOverrides = structuredClone(
+    experiment.metricOverrides || []
+  );
+  for (let i = 0; i < defaultMetricOverrides.length; i++) {
+    for (const key in defaultMetricOverrides[i]) {
+      // fix fields with percentage values
+      if (
+        [
+          "winRisk",
+          "loseRisk",
+          "maxPercentChange",
+          "minPercentChange",
+        ].includes(key)
+      ) {
+        defaultMetricOverrides[i][key] *= 100;
+      }
+    }
+  }
```

Comment:
Having to iterate and multiply/divide by 100 is a little unfortunate but I see the complication we're dealing with ... Is it necessary just so we can display the values (and process input) properly? Or are there other calculations happening?

Reply:
I know, this isn't my favorite way of doing things. Unfortunately though because of the useForm hooks, decoupling a field's stored value from its presentation (by a factor of 100) doesn't work out of the box. We'd need to do more work to update the field integrations, which I think is a bit outside this PR.

Reply:
That's what I concluded as well ... Makes sense, thanks for clarifying